platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/foreign/entity/input/browse/EmfEntityInputBrowseTable.java

```diff
@@ -0,0 +1,33 @@
+package com.softicar.platform.emf.attribute.field.foreign.entity.input.browse;
+
+import com.softicar.platform.common.container.data.table.IDataTableColumn;
+import com.softicar.platform.common.container.data.table.in.memory.AbstractInMemoryDataTable;
+import com.softicar.platform.dom.elements.input.auto.IDomAutoCompleteInputEngine;
+import com.softicar.platform.emf.EmfI18n;
+import com.softicar.platform.emf.entity.IEmfEntity;
+
+class EmfEntityInputBrowseTable<E extends IEmfEntity<E, ?>> extends AbstractInMemoryDataTable<E> {
+
+	private final IDomAutoCompleteInputEngine<E> inputEngine;
+	private final IDataTableColumn<E, String> nameColumn;
+
+	public EmfEntityInputBrowseTable(IDomAutoCompleteInputEngine<E> inputEngine) {
+
+		this.inputEngine = inputEngine;
```

Comment:
```suggestion
		this.inputEngine = Objects.requireNonNull(inputEngine);
```
...again, to fail early.